profile_queries.sh

```diff
@@ -127,6 +173,20 @@ generate_random_schema() {
     done
 }
 
+# Function to check whether the query has the word 'PROFILE' in it
+has_profile() {
+    # Execute the query and capture the output
+    OUTPUT=$(vsql -c "SELECT request FROM query_requests WHERE transaction_id=$TXN_ID AND statement_id=$STMT_ID AND request ILIKE 'PROFILE%';")
```

Comment:
While this will function correctly, better to use vsql -Atc and not just -c. AND why are you not using $VSQL? If the user has a password, this will bomb out as-is I believe.